{
  "_from": "assert-error",
  "_id": "assert-error@1.0.3",
  "_inBundle": false,
  "_integrity": "sha1-ReesJZtc7gOVOthHnEeNm0zLNBs=",
  "_location": "/assert-error",
  "_phantomChildren": {},
  "_requested": {
    "type": "tag",
    "registry": true,
    "raw": "assert-error",
    "name": "assert-error",
    "escapedName": "assert-error",
    "rawSpec": "",
    "saveSpec": null,
    "fetchSpec": "latest"
  },
  "_requiredBy": [
    "#USER",
    "/"
  ],
  "_resolved": "https://registry.npmjs.org/assert-error/-/assert-error-1.0.3.tgz",
  "_shasum": "45e7ac259b5cee03953ad8479c478d9b4ccb341b",
  "_spec": "assert-error",
  "_where": "/Users/thomasfuchs/Projects/test/learn-blog/server",
  "author": {
    "name": "Sergii Iefremov",
    "email": "iefserge@runtimejs.org"
  },
  "bugs": {
    "url": "https://github.com/iefserge/assert-error/issues"
  },
  "bundleDependencies": false,
  "deprecated": false,
  "description": "Basic assert function that throws provided error object",
  "devDependencies": {
    "tape": "^4.0.0"
  },
  "homepage": "https://github.com/iefserge/assert-error",
  "keywords": [
    "assert",
    "error"
  ],
  "license": "Apache-2.0",
  "main": "index.js",
  "name": "assert-error",
  "repository": {
    "type": "git",
    "url": "git+ssh://git@github.com/iefserge/assert-error.git"
  },
  "scripts": {
    "test": "tape test.js"
  },
  "version": "1.0.3"
}
